codexctl/sync.py

```diff
@@ -147,6 +155,16 @@ def download(self, document, location="", overwrite=False):
             print(f"Error trying to download {filename}: {error}")
             return False
 
+    def __is_newer(self, document, fileLocation):
+        import time

```

Comment:
Why is this online instead of at the top of the file?

Reply:
Feels natural to keep related code together as their logic flows one after another.

Put another way, `__is_newer()` is only ever used from `download()`, and is just a part of the original function but extracted. When reading, the code fits in a single screen, and the sequence of operations matches the visual layout of code.

Reply:
But it's not the best practice for python to have imports inline. Inline imports can happen, but are meant for exceptional circumstances.

If the only reason is that you prefer it that way, this codebase is not written to your preferences, you should be matching the existing style and practices of the codebase.

Reply:
My bad, I misunderstood your question; I thought you were asking to move the whole function somewhere else, and not only the import.

I run it through isort just now.